src/app/apiAndObjects/api/api.service.ts

```diff
@@ -29,6 +30,7 @@ export class ApiService extends BaseApi<DictionaryType> {
   // eslint-disable-next-line @typescript-eslint/member-ordering
   public readonly currentData = {
     getMicronutrientDataOptions: new GetMicronutrientDataOptions(this.USE_LIVE_API),
+    getSubRegionData: new GetSubRegionData(this.USE_LIVE_API),
   };
   // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
what does the ESLint rule do thats not acceptable re: ordering?

Reply:
ah, so its wanting to have `public` followed by `private`.

Reply:
Yeah.  I don't want to disable the rule but in this case it's nice to order things in a specific way.

Reply:
does it impact the logic or is it Personal Preference to not use the Angular standard?https://angular.io/guide/styleguide#member-sequence

Reply:
It's more a proximity thing.  It being close to the configure() function is a reminder that if you create a new group it needs to be registered in there.

I'm just having a look at reworking now...

I'm all for the eslint rules, but judging by the number of "// eslint-disable" lines in the codebase at the moment, the current rule set might need some massaging before it's a hassle free process.

Reply:
Re: proximity - agree it makes sense, although a comment block/header may work as replacement advising others to extend/change

I'd not spotted other disabling (other than svea's PR today), but thats an oversight of the reviewers (thats me missing some too!) not spotting. If there are ESLint rules which are not working we should raise them with rest of devs to decide if its bikeshedding preferences or not.

Reply:
Made an issue to store these: https://github.com/micronutrientsupport/micronutrient-support-tool/issues/87

Reply:
There are a couple of API files that I have to yurn off formatOnSave for otherwise the formatting fails the eslinting. :-(

Reply:
Changes pushed.  Have another look @Fuhji